packages/core/runtime/src/types/NullableType.ts

```diff
@@ -21,6 +22,13 @@ export default class NullableType extends Type {
   ) {
     return this.nullableType.deriveLiteral(annotations, context);
   }
+  public async validate(value: unknown, context: MantaStyleContext) {
+    return (
+      this.type.validate(value, context) ||
+      MantaStyle.UndefinedKeyword.validate(value) ||
+      MantaStyle.NullKeyword.validate(value)
```

Comment:
how come .validate can be used as `static`?

Reply:
`UndefinedKeyword`, `NullKeyword` and other keywords are instances.